src/replace/src/replace.tcl

```diff
@@ -306,3 +306,39 @@ proc global_placement_plot { args } {
   gpl::set_plot_path_cmd $args
 }
 
+
+proc get_global_placement_uniform_density { args } {
```

Comment:
this (and global_placement_debug) should probably be in the gpl namespace rather than global.
they are not for user consumption.

Reply:
ok. I've applied namespace as you suggested.